File: demo/src/main/java/com/example/demo/entity/Food.java
package com.example.demo.entity;

import javax.persistence.*;
import java.math.BigDecimal;


@Entity
public class Food extends BaseEntity {

    private String category;
    private String name;

    private BigDecimal price;

    @Lob
    private String description;

    private String imagePath;

    private Integer quantity;

    public Food() {
    }

    public String getCategory() {
        return category;
    }

    public void setCategory(String category) {
        this.category = category;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public BigDecimal getPrice() {
        return price;
    }

    public void setPrice(BigDecimal price) {
        this.price = price;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public String getImagePath() {
        return imagePath;
    }

    public void setImagePath(String imagePath) {
        this.imagePath = imagePath;
    }

    public Integer getQuantity() {
        return quantity;
    }

    public void setQuantity(Integer quantity) {
        this.quantity = quantity;
    }
}

File: food-guro-client/src/app/classes/food.ts

export class Food {
    id: number;
    category: string;
    name: string;
    price: number;
    description: string;
    imagePath: string;
    quantity: number;
    public constructor(id: number, category: string, name: string, price: number, description: string, imagePath: string, quantity: number) {
        this.id = id;
        this.category = category;
        this.name = name;
        this.price = price;
        this.description = description;
        this.imagePath = imagePath;
        this.quantity = quantity;
    }


}